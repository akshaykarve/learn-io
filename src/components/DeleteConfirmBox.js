import React from 'react';
import './ComponentStyle.css';

const DeleteConfirmBox=({deletePlatform,onDeletePlatform,onChangeDelete})=>{
	return (
		<section id="overlay">
			<div className='overlayStyle'>
				<div className='deleteConfirm'>
					<h2>Are you sure you want to delete</h2>
					<h2>"{deletePlatform.platformName}"?</h2>
					<button className='deleteButtonStyle' onClick={()=>onDeletePlatform(deletePlatform)}>Yes</button>
					<button className='deleteButtonStyle' onClick={()=>onChangeDelete('',false)}>No</button>
				</div>
			</div>
		</section>
	);
}
export default DeleteConfirmBox;