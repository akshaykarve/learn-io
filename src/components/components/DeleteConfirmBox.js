import React from 'react';
import '../ComponentStyle.css';

const DeleteConfirmBox=({deletePlatform,onDeletePlatform,setDeletePlatform})=>{
	if(deletePlatform==='') return null
	return (
		<section id="overlay">
			<div className='overlayStyle'>
				<div className='deleteConfirm'>
					<h2>Are you sure you want to delete</h2>
					<h2>"{deletePlatform.platformName}"?</h2>
					<button className='deleteButtonStyle' onClick={()=>onDeletePlatform(deletePlatform)}>Yes</button>
					<button className='deleteButtonStyle' onClick={()=>setDeletePlatform('')}>No</button>
				</div>
			</div>
		</section>
	);
}

export default DeleteConfirmBox;