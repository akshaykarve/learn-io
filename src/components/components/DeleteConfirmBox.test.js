import React from 'react';
import renderer from 'react-test-renderer';
import {default as DeleteConfirmBox} from "./DeleteConfirmBox";

it('Delete Confirm Box', () => {
	
	const tree = renderer.create(
		<DeleteConfirmBox deletePlatform={{platformName:"test"}}
		/>
	).toJSON();
	expect(tree).toMatchSnapshot();
});
