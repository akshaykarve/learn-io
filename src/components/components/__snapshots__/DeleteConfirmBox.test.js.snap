// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Delete Confirm Box 1`] = `
<section
  id="overlay"
>
  <div
    className="overlayStyle"
  >
    <div
      className="deleteConfirm"
    >
      <h2>
        Are you sure you want to delete
      </h2>
      <h2>
        "
        test
        "?
      </h2>
      <button
        className="deleteButtonStyle"
        onClick={[Function]}
      >
        Yes
      </button>
      <button
        className="deleteButtonStyle"
        onClick={[Function]}
      >
        No
      </button>
    </div>
  </div>
</section>
`;
